xcode/Subconscious/Shared/Components/Notebook/NotebookNavigationView.swift

```diff
@@ -124,7 +124,8 @@ struct NotebookNavigationView: View {
                         Button(
                             action: {
                                 let detail = UserProfileDetailDescription(
-                                    address: Slashlink(petname: Petname("ben")!).toPublicMemoAddress()
+                                    user: Petname("flubbo")!,
+                                    spherePath: [Petname("flubbo")!]
```

Comment:
Remove this, but we will ultimately need a way to open this view for the sphere owner... who has no petname 🤔